tests/utils.py

```diff
@@ -681,6 +685,46 @@ def variant2(cls, session: "Session") -> DataFrame:
         )
         return df.select(parse_json("values").as_("src"))
 
+    @classmethod
+    def datetime_primitives1(cls, session: "Session") -> DataFrame:
+        data = [
+            (
+                1706774400.987654321,
+                1706774400,
+                "2024-02-01 00:00:00.000000",
```

Comment:
can we add one more date that is using the string format but also includes timezone info? or is it supported by snowflake?